### PR TITLE
Add tooltip JS

### PR DESCRIPTION
**Pull request by deepaklalwani97**
- Adds tooltip JS used in author details module on single pages.

### CODE REVIEW

**Review by yahilmadakiya — APPROVED** [review]

File: packages/larva-js/src/interface/Tooltip/Tooltip.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
+/**
+ * JS class to create/handle tooltips
```
Comment (laras126) [inline packages/larva-js/src/interface/Tooltip/Tooltip.js:2]: Add details here about the required markup and any required CSS for using this feature - e.g. are there any specific patterns that it works well with, or how the classes should be added to the DOM. Thanks!!

**Review by laras126 — CHANGES_REQUESTED** [review]
Looks great! Just need to add some documentation about how to use it.

File: packages/larva-js/src/interface/Tooltip/Tooltip.js (hunk 1 above)
Comment (deepaklalwani97) [reply packages/larva-js/src/interface/Tooltip/Tooltip.js:2]: Hey @laras126, Thanks I have updated the PR to add the documentation for the script let me know if any change required.

File: packages/larva-js/src/interface/Tooltip/Tooltip.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+ *
+ * * The required selectors for this script are:
+ * - .js-Tooltip-parent - Added to the parent of tooltip which opens the tooltip on hover/focus
+ * - .js-Tooltip - Added to the container of the actual tooltip which is displayed on parents hover/focus.
+ *
+ * Example:
+ * <button class="js-Tooltip-parent">
+ * Tooltip trigger
+ * <div class="js-tooltip">Tooltip content</div>
```
Comment (laras126) [inline packages/larva-js/src/interface/Tooltip/Tooltip.js:11]: Update casing on `js-tooltip` here – otherwise looks great!

Comment (deepaklalwani97) [reply packages/larva-js/src/interface/Tooltip/Tooltip.js]: Thanks, I have updated.

**Review by laras126 — APPROVED** [review]
Looks great!

**Pull request merged**